src/index.js

```diff
@@ -1,34 +1,105 @@
 import './style.css';
 
 const list = document.querySelector('.list');
+const inputTodo = document.querySelector('.add-todo');
+const addTodoBtn = document.querySelector('.submit');
 
-const todoList = [
-  {
-    id: 0,
-    do: 'wash the dishes',
-    boolean: false,
-  },
-  {
-    id: 1,
-    do: 'complete to do list project',
-    boolean: false,
-  },
-  {
-    id: 2,
-    do: 'fix care',
-    boolean: false,
-  },
-];
-
-for (let i = 0; i < todoList.length; i += 1) {
-  list.innerHTML = ` ${list.innerHTML}
-        <li class="item">
-        <div class="left-itmes">
-        <input type="checkbox" class="checkbox">
-          <p id=${todoList[i].id} class="text">${todoList[i].do}</p>
-        </div>
-        <i class="fa-solid fa-ellipsis-vertical"></i>
-      </li>
-      <hr>
-        `;
+class TodoTask {
+  constructor(desp, arr, done = false) {
+    this.desp = desp;
+    this.index = arr.length;
+    this.done = done;
+  }
 }
+
+class TaskLisk {
+  constructor() {
+    this.tasks = [];
+  }
+
+  saveTolocal() {
+	  localStorage.setItem('tasks', JSON.stringify(this.tasks));
+  }
+
+getFromLocal = () => {
+  this.tasks = localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
+}
+
+renderTodo = (list) => {
+  list.innerHTML = '';
+  this.tasks.forEach((el, i) => {
+    const item = document.createElement('li');
+    item.classList.add('item');
+    item.id = i;
+    item.innerHTML = `
+          <div class="left-itmes">
+          <input type="checkbox" class="checkbox">
+          <input type='text' class='todo' value='${el.desp}'>
+          </div>
+          <div>
+          <i class="fa-solid fa-ellipsis-vertical icon-dots"></i>
+          <i class="fa-solid fa-trash-can  icon-remove  hidden"></i>
+          </div>
+    `;
+
+    item.addEventListener('click', () => {
+      const iconRemove = item.querySelector('.icon-remove');
+      const iconDots = item.querySelector('.icon-dots');
+      iconRemove.classList.toggle('hidden');
+      iconDots.classList.toggle('hidden');
+      iconRemove.addEventListener('click', (event) => {
+        const taskItem = event.target.parentNode.parentNode;
+        this.removeItem(taskItem);
+      });
+    });
+
+    item.style.borderBottom = '1px solid #aaa';
+
+    const checkBox = item.querySelector('.checkbox');
+    const todo = item.querySelector('.todo');
+    todo.classList.add('todo-style');
+    checkBox.addEventListener('click', () => {
+      todo.classList.toggle('line-through');
+    });
```

Comment:
- [ ] Currently, your task is not marked as `done: true` in the loacalStorage when you click on the checkbox. This brings a major issue, the task does not keep the checked state on page refresh. Let's fix that